### DISCUSSION TITLE
Deleting a user from a group seem not to be working through API

### DISCUSSION THREAD

**Discussion by ghost** [keycloak/keycloak#15698] [Q&A] [answered]
Hi,
Unfortunately, I couldn't use [keycloak-nodejs-admin-client](https://github.com/keycloak/keycloak-nodejs-admin-client) under Nest because of typescript versions etc. So I'm targeting the API myself and particularly one API seem not to be working:

`DELETE /{realm}/users/{id}/groups/{groupId}`

So as I understand, here we delete a user from a group by IDs. It does not seem to work, the user has always the same groups after deletion.

Also I raise the point about the API Doc; As you can see:

![image](https://user-images.githubusercontent.com/10267332/203953802-a1c45850-c1ab-40f1-a5af-f257d89f494f.png)

Some endpoints don't have a description contrary to the others:

![image](https://user-images.githubusercontent.com/10267332/203953952-137f2b6f-61c0-40ea-abff-2e35f374cee4.png)

Can someone confirm the API ?

Thanks a lot

**Comment by DAHAG-ArisNourbakhsh** [comment, marked as answer]
The Admin console uses that particular API endpoint when you are on a specific user and do a `Leave` on any group on their `Groups` tab.

(In general every single endpoint that is offered by Keycloak is used in some form or another in the Admin. When I'm unsure about certain API endpoints I F12 the next best Feature in the Admin that does the same thing as I want to do and use that as a sanity check/example)

I tried it on Keycloak 20 in the UI manually and also programmatically and can confirm that it does indeed work. 
Also the endpoint would return 400/500 range status codes when: userId cannot be found, group id cannot be found and if the authenticated thingy doesn't have the necessary realm permissions.

**Comment by ghost** [comment]
my bad. it does work. probably my Uri was incorrect or my client js code. 
thank you for the support 🙂
